### ISSUE TITLE
Python IndexError: list index out of range

### ISSUE THREAD

**Issue by monkTertman** [PAhelper/PhoenixAdult.bundle#966]
**Describe the bug**

**Include**
1. Scene URL 
2. File Name `MassageCreep - Karina White - Erotic Stone Massage.mp4`
3. Log File

```
2020-08-26 00:09:19,911 (d60) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IwczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGU2CnMxNgo2NWI5OGYwZjlmZTNiY2ZlczE3Cm9wZW5TdWJ0aXRsZXNIYXNoczQ2Ck1hc3NhZ2VjcmVlcCBLYXJpbmEgV2hpdGUgRXJvdGljIFN0b25lIE1hc3NhZ2VzNApuYW1lczEzNgpYJTNBJTVDSGltYW5rJTVDU29mdHdhcmVzJTVDbmV0d29yayU1Q1JFQ1lDTEVSJTVDazRyaW5hJTIwd2hpdDMlNUNNYXNzYWdlQ3JlZXAlMjAtJTIwS2FyaW5hJTIwV2hpdGUlMjAtJTIwRXJvdGljJTIwU3RvbmUlMjBNYXNzYWdlJTJFbXA0czgKZmlsZW5hbWVzNDAKNWFlNzk5MGIzMzU2NWI0ODQzY2I2NTNlYTIyOTc4MWNjYWJiOWQ3NnM4CnBsZXhIYXNoczcKMTkxMTU3NnM4CmR1cmF0aW9uczQKMjExNHMyCmlkcjAK
2020-08-26 00:09:19,913 (d60) :  DEBUG (runtime:49) - Received packed state data (80 bytes)
2020-08-26 00:09:19,914 (d60) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IwczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGU2CnMxNgo2NWI5OGYwZjlmZTNiY2ZlczE3Cm9wZW5TdWJ0aXRsZXNIYXNoczQ2Ck1hc3NhZ2VjcmVlcCBLYXJpbmEgV2hpdGUgRXJvdGljIFN0b25lIE1hc3NhZ2VzNApuYW1lczEzNgpYJTNBJTVDSGltYW5rJTVDU29mdHdhcmVzJTVDbmV0d29yayU1Q1JFQ1lDTEVSJTVDazRyaW5hJTIwd2hpdDMlNUNNYXNzYWdlQ3JlZXAlMjAtJTIwS2FyaW5hJTIwV2hpdGUlMjAtJTIwRXJvdGljJTIwU3RvbmUlMjBNYXNzYWdlJTJFbXA0czgKZmlsZW5hbWVzNDAKNWFlNzk5MGIzMzU2NWI0ODQzY2I2NTNlYTIyOTc4MWNjYWJiOWQ3NnM4CnBsZXhIYXNoczcKMTkxMTU3NnM4CmR1cmF0aW9uczQKMjExNHMyCmlkcjAK
2020-08-26 00:09:19,914 (d60) :  INFO (agentkit:961) - Searching for matches for {'openSubtitlesHash': '65b98f0f9fe3bcfe', 'name': 'Massagecreep Karina White Erotic Stone Massage', 'filename': 'X%3A%RECYCLER%5Ck4rina%20whit3%5CMassageCreep%20-%20Karina%20White%20-%20Erotic%20Stone%20Massage%2Emp4', 'plexHash': '5ae7990b33565b4843cb653ea229781ccabb9d76', 'duration': '1911576', 'id': '2114'}
2020-08-26 00:09:19,914 (d60) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/library/metadata/2114/tree'
2020-08-26 00:09:19,924 (d60) :  INFO (logkit:16) - *******MEDIA TITLE****** Massagecreep Karina White Erotic Stone Massage
2020-08-26 00:09:19,924 (d60) :  INFO (logkit:16) - Getting Search Settings for: Massagecreep Karina White Erotic Stone Massage
2020-08-26 00:09:19,924 (d60) :  INFO (logkit:16) - mediaTitle w/ possible abbreviation: Massagecreep Karina White Erotic Stone Massage
2020-08-26 00:09:19,941 (d60) :  INFO (logkit:16) - mediaTitle w/ possible abbrieviation fixed: Massagecreep Karina White Erotic Stone Massage
2020-08-26 00:09:19,947 (d60) :  INFO (logkit:16) - Site found with method #3
2020-08-26 00:09:19,947 (d60) :  INFO (logkit:16) - ^^^^^^^ siteID: 311
2020-08-26 00:09:19,947 (d60) :  INFO (logkit:16) - ^^^^^^^ Shortening Title
2020-08-26 00:09:19,948 (d60) :  INFO (logkit:16) - massagecreep vs massagecreep
2020-08-26 00:09:19,948 (d60) :  INFO (logkit:16) - 1
2020-08-26 00:09:19,950 (d60) :  INFO (logkit:16) - 8
2020-08-26 00:09:19,950 (d60) :  INFO (logkit:16) - searchTitle (before date processing): Karina White Erotic Stone Massage
2020-08-26 00:09:19,950 (d60) :  INFO (logkit:16) - Search Title: Karina White Erotic Stone Massage
2020-08-26 00:09:19,950 (d60) :  INFO (logkit:16) - Karina%20White%20Erotic%20Stone%20Massage
2020-08-26 00:09:19,953 (d60) :  INFO (logkit:16) - Requesting GET "https://pornpros.com/video/karina-white-erotic-stone-massage"
2020-08-26 00:09:20,053 (1904) :  INFO (logkit:16) - Requesting GET "https://cdnh1.letsdoeit.com/photo/crop/512x288/5/7/1/0/4657-wb-landscape-hardcore-pavzhz8fb12f3d61.jpg"
2020-08-26 00:09:20,734 (d60) :  CRITICAL (core:574) - Exception in the search function of agent named 'PhoenixAdult', called with keyword arguments {'openSubtitlesHash': '65b98f0f9fe3bcfe', 'name': 'Massagecreep Karina White Erotic Stone Massage', 'filename': 'X%3A%5CRECYCLER%5Ck4rina%20whit3%5CMassageCreep%20-%20Karina%20White%20-%20Erotic%20Stone%20Massage%2Emp4', 'plexHash': '5ae7990b33565b4843cb653ea229781ccabb9d76', 'duration': '1911576', 'id': '2114'} (most recent call last):
  File "C:\Program Files (x86)\Plex\Plex Media Server\Resources\Plug-ins-b23ab3896\Framework.bundle\Contents\Resources\Versions\2\Python\Framework\api\agentkit.py", line 1011, in _search
    agent.search(*f_args, **f_kwargs)
  File "C:\Users\Monk\AppData\Local\Plex Media Server\Plug-ins\PhoenixAdult.bundle\Contents\Code\__init__.py", line 858, in search
    results = PAsearchSites.siteAussieAss.search(results, encodedTitle, searchTitle, siteNum, lang, searchDate)
  File "C:\Users\Monk\AppData\Local\Plex Media Server\Plug-ins\PhoenixAdult.bundle\Contents\Code\networkPornPros.py", line 15, in search
    titleNoFormatting = detailsPageElements.xpath('//h1')[0].text_content()
  File "C:\Program Files (x86)\Plex\Plex Media Server\Resources\Plug-ins-b23ab3896\Framework.bundle\Contents\Resources\Versions\2\Python\Framework\code\sandbox.py", line 108, in <lambda>
    _getitem_           = lambda x, y: x.__getitem__(y),
IndexError: list index out of range

2020-08-26 00:09:20,736 (d60) :  DEBUG (runtime:88) - Sending packed state data (112 bytes)
2020-08-26 00:09:20,736 (d60) :  DEBUG (runtime:924) - Response: [200] str, 16 bytes
```

Another log:
```
2020-08-26 02:53:18,092 (14c4) :  INFO (agentkit:961) - Searching for matches for {'openSubtitlesHash': '50b84e428a8f939a', 'name': 'Puremature Kendra Lust Prefers Amazing Sex With Young Men', 'filename': 'X%3A%5CRECYCLER%5Ckendra%5CPureMature%20Kendra%20Lust%20Prefers%20Amazing%20Sex%20With%20Young%20Men%2Emp4', 'plexHash': 'ee181d1be0257160327738c1cd7cfd1068b0045a', 'duration': '1568880', 'id': '2201'}
2020-08-26 02:53:18,092 (14c4) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/library/metadata/2201/tree'
2020-08-26 02:53:18,101 (14c4) :  INFO (logkit:16) - *******MEDIA TITLE****** Puremature Kendra Lust Prefers Amazing Sex With Young Men
2020-08-26 02:53:18,101 (14c4) :  INFO (logkit:16) - Getting Search Settings for: Puremature Kendra Lust Prefers Amazing Sex With Young Men
2020-08-26 02:53:18,101 (14c4) :  INFO (logkit:16) - mediaTitle w/ possible abbreviation: Puremature Kendra Lust Prefers Amazing Sex With Young Men
2020-08-26 02:53:18,115 (14c4) :  INFO (logkit:16) - mediaTitle w/ possible abbrieviation fixed: Puremature Kendra Lust Prefers Amazing Sex With Young Men
2020-08-26 02:53:18,121 (14c4) :  INFO (logkit:16) - Site found with method #3
2020-08-26 02:53:18,121 (14c4) :  INFO (logkit:16) - ^^^^^^^ siteID: 484
2020-08-26 02:53:18,121 (14c4) :  INFO (logkit:16) - ^^^^^^^ Shortening Title
2020-08-26 02:53:18,121 (14c4) :  INFO (logkit:16) - puremature vs puremature
2020-08-26 02:53:18,122 (14c4) :  INFO (logkit:16) - 1
2020-08-26 02:53:18,122 (14c4) :  INFO (logkit:16) - 8
2020-08-26 02:53:18,122 (14c4) :  INFO (logkit:16) - searchTitle (before date processing): Kendra Lust Prefers Amazing Sex With Young Men
2020-08-26 02:53:18,124 (14c4) :  INFO (logkit:16) - Search Title: Kendra Lust Prefers Amazing Sex With Young Men
2020-08-26 02:53:18,124 (14c4) :  INFO (logkit:16) - Kendra%20Lust%20Prefers%20Amazing%20Sex%20With%20Young%20Men
2020-08-26 02:53:18,127 (14c4) :  INFO (logkit:16) - Requesting GET "https://puremature.com/video/kendra-lust-prefers-amazing-sex-with-young-men"
2020-08-26 02:53:19,694 (2c80) :  INFO (logkit:16) - Genres
2020-08-26 02:53:19,697 (2c80) :  INFO (logkit:16) - Actors
2020-08-26 02:53:19,697 (2c80) :  INFO (logkit:16) - Actor: Alina Lopez https://images04-openlife.gammacdn.com/actors/51318/51318_200x300.jpg
2020-08-26 02:53:19,697 (2c80) :  INFO (logkit:16) - Actor: Danny Mountain https://images02-openlife.gammacdn.com/actors/31460/31460_200x300.jpg
2020-08-26 02:53:19,776 (2a48) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlVwZGF0ZU1ldGFkYXRh/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMTAKczIKZW5zNApsYW5nYjFzNQpmb3JjZWIwczgKcGVyaW9kaWNzNAoxNTc0czQKZGJpZGkwCnM3CnZlcnNpb25uczEwCnBhcmVudEdVSURuczgKcGFyZW50SURzNQpNb3ZpZXMxMAptZWRpYV90eXBlczU4CmNvbS5wbGV4YXBwLmFnZW50cy5waG9lbml4YWR1bHQ6Ly80MTI3ODMxfDYyfHNjZW5lP2xhbmc9ZW5zNApndWlkczE2CjQxMjc4MzF8NjJ8c2NlbmVzMgppZHIwCg__
2020-08-26 02:53:19,778 (2a48) :  DEBUG (runtime:49) - Received packed state data (80 bytes)
2020-08-26 02:53:19,779 (2a48) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlVwZGF0ZU1ldGFkYXRh/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMTAKczIKZW5zNApsYW5nYjFzNQpmb3JjZWIwczgKcGVyaW9kaWNzNAoxNTc0czQKZGJpZGkwCnM3CnZlcnNpb25uczEwCnBhcmVudEdVSURuczgKcGFyZW50SURzNQpNb3ZpZXMxMAptZWRpYV90eXBlczU4CmNvbS5wbGV4YXBwLmFnZW50cy5waG9lbml4YWR1bHQ6Ly80MTI3ODMxfDYyfHNjZW5lP2xhbmc9ZW5zNApndWlkczE2CjQxMjc4MzF8NjJ8c2NlbmVzMgppZHIwCg__
2020-08-26 02:53:19,779 (2a48) :  DEBUG (model:32) - Loading model with GUID com.plexapp.agents.phoenixadult://4127831|62|scene?lang=en
2020-08-26 02:53:19,779 (2a48) :  ERROR (model:205) - Cannot read model from C:\Users\Monk\AppData\Local\Plex Media Server\Metadata\Movies\1\8d42f8166064fb273423bdebd9c8a2b53c8ef7d.bundle\Contents\com.plexapp.agents.phoenixadult
2020-08-26 02:53:19,782 (2a48) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/library/metadata/1574/tree'
2020-08-26 02:53:19,798 (2a48) :  INFO (logkit:16) - ******UPDATE CALLED*******
2020-08-26 02:53:19,799 (2a48) :  INFO (logkit:16) - 62
2020-08-26 02:53:19,801 (2a48) :  INFO (logkit:16) - Requesting GET "http://www.brazzers.com"
2020-08-26 02:53:19,812 (14c4) :  CRITICAL (core:574) - Exception in the search function of agent named 'PhoenixAdult', called with keyword arguments {'openSubtitlesHash': '50b84e428a8f939a', 'name': 'Puremature Kendra Lust Prefers Amazing Sex With Young Men', 'filename': 'X%3A%5CRECYCLER%5Ckendra%5CPureMature%20Kendra%20Lust%20Prefers%20Amazing%20Sex%20With%20Young%20Men%2Emp4', 'plexHash': 'ee181d1be0257160327738c1cd7cfd1068b0045a', 'duration': '1568880', 'id': '2201'} (most recent call last):
IndexError: list index out of range

2020-08-26 02:53:19,813 (14c4) :  DEBUG (runtime:88) - Sending packed state data (112 bytes)
2020-08-26 02:53:19,813 (14c4) :  DEBUG (runtime:924) - Response: [200] str, 16 bytes
```

**Comment by DirtyRacer1337** [comment]
Tilte only, rename to `MassageCreep - Erotic Stone Massage`

**Comment by monkTertman** [comment]
@DirtyRacer1337 This is not just happening for this one file.
Its happening for quite a lot others too, like Kink Network which supports enhanced search.

Log:
```
2020-08-27 00:12:16,267 (1c58) :  INFO (agentkit:961) - Searching for matches for {'year': None, 'id': '1536', 'name': 'kink - Abella Danger Day One hardcore anal'}
2020-08-27 00:12:16,269 (1c58) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/library/metadata/1536/tree'
2020-08-27 00:12:16,275 (1c58) :  INFO (logkit:16) - *******MEDIA TITLE****** kink Abella Danger Day One hardcore anal
2020-08-27 00:12:16,275 (1c58) :  INFO (logkit:16) - Getting Search Settings for: kink Abella Danger Day One hardcore anal
2020-08-27 00:12:16,275 (1c58) :  INFO (logkit:16) - mediaTitle w/ possible abbreviation: kink Abella Danger Day One hardcore anal
2020-08-27 00:12:16,280 (1c58) :  INFO (logkit:16) - mediaTitle w/ possible abbrieviation fixed: kink Abella Danger Day One hardcore anal
2020-08-27 00:12:16,283 (1c58) :  INFO (logkit:16) - Site found with method #3
2020-08-27 00:12:16,283 (1c58) :  INFO (logkit:16) - ^^^^^^^ siteID: 490
2020-08-27 00:12:16,283 (1c58) :  INFO (logkit:16) - ^^^^^^^ Shortening Title
2020-08-27 00:12:16,283 (1c58) :  INFO (logkit:16) - kink vs kink
2020-08-27 00:12:16,283 (1c58) :  INFO (logkit:16) - 1
2020-08-27 00:12:16,285 (1c58) :  INFO (logkit:16) - 8
2020-08-27 00:12:16,285 (1c58) :  INFO (logkit:16) - searchTitle (before date processing): Abella Danger Day One hardcore anal
2020-08-27 00:12:16,285 (1c58) :  INFO (logkit:16) - Search Title: Abella Danger Day One hardcore anal
2020-08-27 00:12:16,285 (1c58) :  INFO (logkit:16) - Abella%20Danger%20Day%20One%20hardcore%20anal
2020-08-27 00:12:16,286 (1c58) :  INFO (logkit:16) - Requesting GET "http://www.kink.com/search?q=Abella%20Danger%20Day%20One%20hardcore%20anal"
2020-08-27 00:12:19,792 (1c58) :  CRITICAL (core:574) - Exception in the search function of agent named 'PhoenixAdult', called with keyword arguments {'year': None, 'id': '1536', 'name': 'kink - Abella Danger Day One hardcore anal'} (most recent call last):
  File "C:\Program Files (x86)\Plex\Plex Media Server\Resources\Plug-ins-b23ab3896\Framework.bundle\Contents\Resources\Versions\2\Python\Framework\api\agentkit.py", line 1011, in _search
    agent.search(*f_args, **f_kwargs)
  File "C:\Users\Monk\AppData\Local\Plex Media Server\Plug-ins\PhoenixAdult.bundle\Contents\Code\__init__.py", line 858, in search
    results = PAsearchSites.siteAussieAss.search(results, encodedTitle, searchTitle, siteNum, lang, searchDate)
  File "C:\Users\Monk\AppData\Local\Plex Media Server\Plug-ins\PhoenixAdult.bundle\Contents\Code\networkKink.py", line 30, in search
    shootID = searchResult.xpath('.//span[@class="favorite-button"]/@data-id')[0]
  File "C:\Program Files (x86)\Plex\Plex Media Server\Resources\Plug-ins-b23ab3896\Framework.bundle\Contents\Resources\Versions\2\Python\Framework\code\sandbox.py", line 108, in <lambda>
    _getitem_           = lambda x, y: x.__getitem__(y),
IndexError: list index out of range

2020-08-27 00:12:19,796 (1c58) :  DEBUG (runtime:88) - Sending packed state data (112 bytes)
2020-08-27 00:12:19,796 (1c58) :  DEBUG (runtime:924) - Response: [200] str, 16 bytes
2020-08-27 00:14:01,677 (3384) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGUzCnM0CjE1MzZzMgppZHMzNgpLaW5rIE5ldHdvcmsgLSBhYmVsbGEgZGFuZ2VyIGRheSBvbmVzNApuYW1lbnM0CnllYXJyMAo_
2020-08-27 00:14:01,677 (3384) :  DEBUG (runtime:49) - Received packed state data (80 bytes)
2020-08-27 00:14:01,678 (3384) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGUzCnM0CjE1MzZzMgppZHMzNgpLaW5rIE5ldHdvcmsgLSBhYmVsbGEgZGFuZ2VyIGRheSBvbmVzNApuYW1lbnM0CnllYXJyMAo_
2020-08-27 00:14:01,680 (3384) :  INFO (agentkit:961) - Searching for matches for {'year': None, 'id': '1536', 'name': 'Kink Network - abella danger day one'}
2020-08-27 00:14:01,680 (3384) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/library/metadata/1536/tree'
2020-08-27 00:14:01,687 (3384) :  INFO (logkit:16) - *******MEDIA TITLE****** Kink Network abella danger day one
2020-08-27 00:14:01,687 (3384) :  INFO (logkit:16) - Getting Search Settings for: Kink Network abella danger day one
2020-08-27 00:14:01,688 (3384) :  INFO (logkit:16) - mediaTitle w/ possible abbreviation: Kink Network abella danger day one
2020-08-27 00:14:01,696 (3384) :  INFO (logkit:16) - mediaTitle w/ possible abbrieviation fixed: Kink Network abella danger day one
2020-08-27 00:14:01,698 (3384) :  INFO (logkit:16) - Site found with method #3
2020-08-27 00:14:01,698 (3384) :  INFO (logkit:16) - ^^^^^^^ siteID: 490
2020-08-27 00:14:01,700 (3384) :  INFO (logkit:16) - ^^^^^^^ Shortening Title
2020-08-27 00:14:01,700 (3384) :  INFO (logkit:16) - kink vs kink
2020-08-27 00:14:01,700 (3384) :  INFO (logkit:16) - 1
2020-08-27 00:14:01,700 (3384) :  INFO (logkit:16) - 8
2020-08-27 00:14:01,700 (3384) :  INFO (logkit:16) - searchTitle (before date processing): Network abella danger day one
2020-08-27 00:14:01,700 (3384) :  INFO (logkit:16) - Search Title: Network abella danger day one
2020-08-27 00:14:01,700 (3384) :  INFO (logkit:16) - Network%20abella%20danger%20day%20one
2020-08-27 00:14:01,703 (3384) :  INFO (logkit:16) - Requesting GET "http://www.kink.com/search?q=Network%20abella%20danger%20day%20one"
2020-08-27 00:14:05,155 (3384) :  CRITICAL (core:574) - Exception in the search function of agent named 'PhoenixAdult', called with keyword arguments {'year': None, 'id': '1536', 'name': 'Kink Network - abella danger day one'} (most recent call last):
  File "C:\Program Files (x86)\Plex\Plex Media Server\Resources\Plug-ins-b23ab3896\Framework.bundle\Contents\Resources\Versions\2\Python\Framework\api\agentkit.py", line 1011, in _search
    agent.search(*f_args, **f_kwargs)
  File "C:\Users\Monk\AppData\Local\Plex Media Server\Plug-ins\PhoenixAdult.bundle\Contents\Code\__init__.py", line 858, in search
    results = PAsearchSites.siteAussieAss.search(results, encodedTitle, searchTitle, siteNum, lang, searchDate)
  File "C:\Users\Monk\AppData\Local\Plex Media Server\Plug-ins\PhoenixAdult.bundle\Contents\Code\networkKink.py", line 30, in search
    shootID = searchResult.xpath('.//span[@class="favorite-button"]/@data-id')[0]
  File "C:\Program Files (x86)\Plex\Plex Media Server\Resources\Plug-ins-b23ab3896\Framework.bundle\Contents\Resources\Versions\2\Python\Framework\code\sandbox.py", line 108, in <lambda>
    _getitem_           = lambda x, y: x.__getitem__(y),
IndexError: list index out of range

2020-08-27 00:14:05,158 (3384) :  DEBUG (runtime:88) - Sending packed state data (112 bytes)
2020-08-27 00:14:05,160 (3384) :  DEBUG (runtime:924) - Response: [200] str, 16 bytes
```

**Comment by DirtyRacer1337** [comment]
Ok, then is definitely on your side. If you really get blocked by provider, you can try to use https://github.com/ValdikSS/GoodbyeDPI, they also have another projects in readme file

**Comment by monkTertman** [comment]
Hey @DirtyRacer1337,
I have setup a VPN and I am able browse all websites now.
But I am still getting this error on this file.
Do you think it's still related to that?

**Comment by DirtyRacer1337** [comment]
That fix should help 4eccdc09b635c96330b25d8c4b23588b1ab656a6

**Comment by monkTertman** [comment]
Thanks man, it works now!
